### PR TITLE
Chore: Put the three doc strings about each op together

### PR DESCRIPTION
**Pull request by jannotti**
This PR should not affect any run-time behavior, nor any generated specs. It simply puts all of the documentation for an opcode together in one place, so it's slightly easier to add opcodes and find all of the documentation about an opcode in the source.

An eventual further change might be to get the documentaion into the OpSpecs themselves, as there is still a little confusion there - immediates are _named_ in the OpSpec, but documented in doc.go.

The "test plan" here is really just that the generated specs are unchanged.

### CODE REVIEW

File: data/transactions/logic/assembler.go [hunk 1]
```diff
@@ -1727,12 +1727,13 @@ func addPseudoDocTags() {
 				continue
 			}
 			msg := fmt.Sprintf("`%s` can be called using `%s` with %s.", spec.Name, name, joinIntsOnOr("immediate", i))
-			desc, ok := opDocByName[spec.Name]
-			if ok {
-				opDocByName[spec.Name] = desc + "<br />" + msg
+			desc := opDescByName[spec.Name]
```
Comment (ahangsu) [inline data/transactions/logic/assembler.go:1730]: I guess the `ok` here goes away for `opDescByName` takes any valid spec name, but previous `opDocByName` accepts a subset of all the valid spec names.

**Review by ahangsu — DISMISSED** [review]
This PR essentially merge following 3 things:
1. `opDocByName`
2. `opDocExtras`
3. `opcodeImmediateNotes`

into one map of `OpDesc`.

File: data/transactions/logic/assembler.go (hunk 1 above)
Comment (jannotti) [reply data/transactions/logic/assembler.go:1730]: The `ok` was checking to see if there was a description in `opDocByName`, that is now equivalent to having a non-blank `.Short` field in `opDescByName`.  So the comparison of `desc.Name` to "==" should be the same.

On the other hand, this code should probably yell and scream if there's no `.Short` description, it probably means the opcode isn't described at all, and that's likely a bug. Let me check it out.

Comment (jannotti) [reply data/transactions/logic/assembler.go:1730]: See latest commit.  Removed the code path that was there only for testing, but was never actually used.

**Review by ahangsu — APPROVED** [review]
👍

**Review by jasonpaulos — APPROVED** [review]
Makes sense

**Pull request merged**